src/mixins/java/org/spongepowered/common/mixin/api/mcp/entity/LivingEntityMixin_API.java

```diff
@@ -74,4 +81,19 @@ public Component getTeamRepresentation() {
         return values;
     }
 
+    @Override
+    public <T extends Projectile> Optional<T> launchProjectile(final EntityType<T> projectileType) {
+        return ProjectileLauncher.launch(checkNotNull(projectileType, "projectileType"), this, null);
```

Comment:
Objects#requireNotNull

(We're trying to cut down on Guava usage if not eliminate it).

Reply:
Oh right I didn't pay attention to it